src/azul/service/responseobjects/elastic_request_builder.py

```diff
@@ -544,7 +545,17 @@ def transform_manifest(self, format: str, filters: JSON):
         request_config_path = "{}/{}".format(config_folder, 'request_config.json')
         request_config = self.open_and_return_json(request_config_path)
         filters = filters['file']
-        manifest_config = request_config['manifest']
+        source_filter = None
+        manifest_config = None
+        entity_type = 'files'
+
+        if format == 'full':
+            source_filter = ['contents.metadata.*']
+            entity_type = 'bundles'
+            manifest_config = self.generate_metadata_manifest()
+
+        manifest_config = request_config['manifest'] if not manifest_config else manifest_config
```

Comment:
I think it's more readable if you just set the variables initially and then overwrite the values in the `if` blocks instead of having a one-liner that sets `manifest_config` to a value if not `manifest_config` else back to itself if it already has a value.
```
manifest_config = request_config['manifest']
source_filter = None
entity_type = 'files'

if format == 'full':
    manifest_config = self.generate_metadata_manifest()
    source_filter = ['contents.metadata.*']
    entity_type = 'bundles'

if format == 'bdbag':
    ...
```

Reply:
Also chain the two if statements where 

```
if format == 'full':
    <code>
elif format == 'bdbag':
    <code>
```